temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsOptions.java

```diff
@@ -44,6 +44,10 @@
   private static final Duration DEFAULT_QUERY_RPC_TIMEOUT = Duration.ofSeconds(10);
   /** Default timeout that will be used to reset connection backoff. */
   private static final Duration DEFAULT_CONNECTION_BACKOFF_RESET_FREQUENCY = Duration.ofSeconds(10);
+  /**
+   * Default timeout that will be used to enter idle channel state and reconnect to temporal server.
+   */
+  private static final Duration DEFAULT_ENTER_IDLE_CHANNEL_STATE_FREQUENCY = Duration.ofMinutes(5);
```

Comment:
I think 5 minutes is too large to wait for rebalancing. What about 1 minute?

Reply:
I was thinking between 1 and 5, went for a potentially less intrusive option. But we can change it to 1 as well.